ansible_collections/arista/avd/docs/getting-started/avd-aap.md

```diff
@@ -0,0 +1,400 @@
+<!--
+  ~ Copyright (c) 2023-2024 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# AVD and Ansible Automation Platform
+
+This guide will walk you through the steps required to get up and running with AVD and Red Hat's Ansible Automation Platform (AAP). AAP is Red Hat's solution to scale automation within an organization, whether by making automation more readily available to team members, adding security capabilities, or simplifying the speed of getting started with Ansible using execution environments.
+
+## Requirements to get started
+
+Please note, that this guide leverages AAP version 2.4. The workflows should be similar in newer versions; if there are any questions, please see the official [AAP documentation](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4).
+
+- An accessible lab topology running Arista EOS.
+- An AVD project or Git repository with playbooks and an inventory. To get started, you may also use any of our [example topologies](../../examples/single-dc-l3ls/README.md).
+- A RHEL instance running AAP.
+  - If you need access to a RHEL instance, you can join the [developer program](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux/developer-program) to get a copy.
+  - To get started, you may also sign up for a 60-day [trial license](https://www.redhat.com/en/technologies/management/ansible/trial) for AAP.
+
+## Topology
+
+Our topology leverages two spines and four leaf nodes to create a layer three leaf spine topology. The topology is managed by Arista CloudVision Portal (CVP). In our case, the EOS nodes are constantly streaming to CVP and gives us the ability to provision the nodes with CVP. AAP will act as our controller to communicate any updates to CVP, which can then push the updates to our topology.
+
+![Topology leveraging Ansible Automation Platform to communicate with CloudVision](../_media/getting-started/aap-avd/aap-cvp-eos.svg)
```

Comment:
For consistency in the document, should we change RH AAP for AAP? And cvp in capital letters (ie CVP)?
Also, the colour of the arrows, does it mean anything? If yes, what? If not, I think to use the same colour will be a good idea

Reply:
I agree with just AAP. Good catch and we should just use CloudVision (CV).

Reply:
Okay leaving in AAP and updating CVP to CV